homeassistant/components/insteon/climate.py

```diff
@@ -202,12 +202,12 @@ async def async_set_temperature(self, **kwargs) -> None:
 
     async def async_set_fan_mode(self, fan_mode: str) -> None:
         """Set new target fan mode."""
-        mode = list(FAN_MODES.keys())[list(FAN_MODES.values()).index(fan_mode)]
+        mode = list(FAN_MODES)[list(FAN_MODES.values()).index(fan_mode)]
```

Comment:
Side note for a future clean up: It's better to define a reverse constant at the module level and look up. This is inefficient.